ansible/roles/common/templates/conf/output/00-local.conf.j2

```diff
@@ -51,6 +58,9 @@
        logstash_format true
        logstash_prefix {{ kibana_log_prefix }}
        flush_interval 15s
+       reconnect_on_error true
+       buffer_type file
+       buffer_path /var/lib/fluentd/data/elasticsearch.buffer/
```

Comment:
This comes from a patch with a -1 - I don't know if the buffering will work properly as the patch doesn't follow the guidelines. I think we should remove that patch.

Reply:
OK, it's gone - for some reason I thought one of the other patches was based on it.